### ISSUE TITLE
Validation / message color change

### ISSUE THREAD

**Issue by philips77** [sberrevoets/SDCAlertView#194]
Hi,
I'm using your lib in a project and would like to add a validation error message.
Right now I'm validating the textField value in shouldDismissHandler from where I would like to show a message in a nice way. My message contains a text "Max n characters" so I was thinking about changing its color to red, or changing the background of the text field. 
The second one I can do fairly easy with ```alert.textFields![0].backgroundColor = UIColor.error```, but the new color get applied only to the inner part of the field, the padding remains white. 
As for the  first one I can do this in a ugly way by getting alert.view?.subviews.first?.subviews[1].subviews[1] and casting it into UILabel. I also tried with ```attributedMessage?.setAttributes([NSForegroundColorAttributeName: UIColor.red], range: NSRange(0 ..< aSubtitle!.characters.count))``` after setting it as a NSMutableAttributedString in the constructor, but this also doesn't seem to work.

How about exposing the message view? I know I can make it on my own with a customView but it would be much easier this way... What do you think?

**Comment by philips77** [comment]
Of course if there are any other ways to show invalid content, like shaking of the alert view, etc, I'd be more than happy. Thanks!

**Comment by sberrevoets** [comment]
In the good old days there was an alert, then later Apple added functionality to show text fields there because they needed it and created a public API for it. Then Apple added other stuff to their alerts and on iOS 6 3rd party developers would too, but there was no official API for it (even though Apple announced there would be).

With iOS 7 this broke, which was why I wrote this project, but realistically Apple should have created a public API for this and gotten rid of the too-specific text field API.

Long story short: text fields shouldn't really have been part of an alert API in the first place, and the only reason I added the same API was to provide feature parity with the official one. But I honestly don't think we should expose more in the public API just for the sake of supporting text fields. If you need more than what is publicly available, I think rolling your own would be your best bet.
